src/components/Cards/CreateAccount.vue

```diff
@@ -4,10 +4,16 @@
       <v-toolbar-title>{{title}}</v-toolbar-title>
     </v-toolbar>
     <v-card-text>
+      <p>
+        Wikibase.cloud is in a closed beta. If you'd like to receive an invite code,
+        <a href="https://lime.wikimedia.de/index.php/717538">
```

Comment:
I think we should add `target="_blank"` here to open the link in a new tab. And also in other places with links (line 85 and 86 also)

Reply:
the AC does not mention that. perhaps we should ask Charlie what she think about that.

Reply:
As I understood Charlie, this is ok as is.